test/integration/query-interface.test.js

```diff
@@ -666,7 +664,10 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           throw new Error('Error not thrown...');
         } catch (error) {
           expect(error).to.be.instanceOf(Sequelize.UnknownConstraintError);
-          expect(error.table).to.equal('users');
+          if (dialect !== 'ibmi') {
+            expect(error.table).to.equal('users');
```

Comment:
Anything we can expect for ibmi?

Reply:
Unfortunately, IBM i doesn't give much good information in its error in this case. Instead, it will simply indicate that there was an unknown constraint of `unknown__constraint__name`.